backbone.layoutmanager.js

```diff
@@ -262,11 +262,6 @@ var LayoutManager = Backbone.View.extend({
     return this.__manager__.renderDeferred.promise();
   },
 
-  // Proxy `then` for easier invocation.
-  then: function() {
-    return this.promise().then.apply(this, arguments);
```

Comment:
Yeah, I suppose you lose that, but you gain less overall changes.  Either way we'll need a minor bump.